molecule/default/prepare.yml

```
@@ -19,3 +19,11 @@
     - name: Update apt cache
       become: yes
       command: apt-get update
+
+- hosts: controllers
+  tasks:
+    - name: Set masquerade rule
+      become: yes
+      ansible.builtin.shell: |
+        iptables -t nat -A POSTROUTING -s  10.96.250.0/24  -o ens3 -j MASQUERADE
+        iptables -t nat -A POSTROUTING -s  10.96.250.0/24  -o ens4 -j MASQUERADE
```

```suggestion
      ansible.builtin.iptables:
        table: nat
        chain: POSTROUTING
        source: 10.96.250.0/24
        out_interface: "{{ ansible_default_ipv4.interface }}"
        jump: MASQUERADE
```

This will be cleaner overall since it will create it only if needed, and also it will use the default interface (with default route) and masquerade that only instead.